dev-docs/publisher-api-reference-old.md

```diff
@@ -14,6 +14,9 @@ This page has documentation for the pre-1.0 public API methods of Prebid.js.
 {: .alert.alert-danger :}
 Warning: do not use this API reference for new Prebid.js implementations. This document is kept for users on old (Pre-1.0) versions of Prebid.js who need to maintain their pages.
 
+{: .alert.alert-danger :}
```

Comment:
Is the plan to remove this page entirely on 9/27?

Reply:
Rather than remove the page which would result in 404s, I'd rather remove all links to it. Then I could remove the content from this page and just put a note on it saying it's been deprecated, and link to the current docs.